libraries/aws_backend.rb

```diff
@@ -172,6 +172,26 @@ def sts_client
   def ssm_client
     aws_client(Aws::SSM::Client)
   end
+
+  def apigateway_client
+    aws_client(Aws::APIGateway::Client)
+  end
+
+  def applicationautoscaling_client
+    aws_client(Aws::ApplicationAutoScaling::Client)
+  end
+
+  def athena_client
+    aws_client(Aws::Athena::Client)
+  end
+
+  def autoscaling_client
+    aws_client(Aws::AutoScaling::Client)
+  end
+
+  def batch_client
+    aws_client(Aws::Batch::Client)
+  end
```

Comment:
why do we need this?

Reply:
These are required when we will be writing the other resources?

Reply:
Okay, please remove them if it is not relevant to this PR..

Reply:
OK